packages/suite/src/views/onboarding/steps/Recovery/index.tsx

```diff
@@ -138,21 +140,13 @@ export const RecoveryStep = () => {
                 key={status} // to properly rerender in translation mode
                 heading={<Translation id="TR_RECOVER_YOUR_WALLET_FROM" />}
                 deviceModel={deviceModel}
-                innerActions={
-                    <OnboardingButtonCta
-                        onClick={() => TrezorConnect.cancel()}
-                        icon="CANCEL"
-                        variant="danger"
-                    >
-                        <Translation id="TR_ABORT" />
-                    </OnboardingButtonCta>
-                }
                 description={pickByDeviceModel(deviceModel, {
                     default: undefined,
                     [DeviceModel.T1]: getModel1Description(),
                     [DeviceModel.TT]: <Translation id="TR_RECOVER_SUBHEADING_TOUCH" />,
                     [DeviceModel.TR]: <Translation id="TR_RECOVER_SUBHEADING_BUTTONS" />,
                 })}
+                isActionAbortable
             >
                 <SelectRecoveryWord />
             </RecoveryStepBox>
```

Comment:
I know it's not part of this issue and it's a nitpick, but I just noticed that `@onboarding/recovery/back-button` is visible and active together with Abort/Cancel button while confirming seed recovery and that back button move back just the UI but not cancel the action on device so it leads to inconsistent state. I believe the back button should be hidden completely in that state.

![image](https://user-images.githubusercontent.com/3729633/233361537-a52ff671-b822-41d7-85f2-8e9448d711c7.png)

Reply:
2285664957d60561b116c8177c26800608e39ce8